app/experimenter/legacy-ui/rapid/components/experiments/ResultsTable.tsx

```diff
@@ -0,0 +1,270 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+
+import {
+  SIGNIFICANCE_TIPS,
+  RESULT_COLUMN_TIPS,
+} from "experimenter-rapid/components/experiments/tooltips";
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import { AnalysisPoint, ExperimentData } from "experimenter-types/experiment";
+
+enum Significance {
+  POSITIVE,
+  NEGATIVE,
+  NEUTRAL,
+}
+
+// A mapping of metric label to relevant statistic. This is
+// used to see which statistic will be used for each metric.
+const resultsMetricsMap = {
+  retained: "binomial",
+  search_count: "mean",
+  identity: "count",
+};
+
+const setPopulationPercentages = (results, totalPopulation) => {
+  Object.keys(results).forEach((branch) => {
+    const populationCount = results[branch]["identity"]["point"];
+    const populationPercent = Math.round(
+      (populationCount / totalPopulation) * 100,
+    );
+    results[branch]["identity"]["percent"] = populationPercent;
+  });
+};
```

Comment:
Same here, could this be done on the backend?

Reply:
yep. moved